app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/AppointmentPending.java

```diff
@@ -1,27 +1,70 @@
 package com.triadss.doctrack2.activity.patient.fragment;
 
 import androidx.fragment.app.Fragment;
-import androidx.fragment.app.FragmentTransaction;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
-import android.annotation.SuppressLint;
+import android.app.Activity;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Toast;
 
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 import com.triadss.doctrack2.R;
+import com.triadss.doctrack2.dto.AppointmentDto;
+import com.triadss.doctrack2.repoositories.AppointmentRepository;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class AppointmentPending extends Fragment {
 
+
+    RecyclerView recyclerView;
+    private AppointmentRepository appointmentRepository;
     private BottomNavigationView bottomNavigationView, PatientbottomNavigationView;
 
+   ArrayList<String> courseName = new ArrayList<>(Arrays.asList("Data Structure"));
```

Comment:
This is an appointment Pending page, please use appropriate variable names.
Also why is there Data Structure as its initial data?

Reply:
Changed {courseName} to {Pending}